src/Microsoft.Diagnostics.Monitoring.RestServer/MetricsOptions.cs

```diff
@@ -51,5 +52,15 @@ public MetricsOptions()
         public int UpdateIntervalSeconds { get; set; }
```

Comment:
Do we also want to allow sampling intervals for different providers?

Reply:
I'm tracking this as a separate issue, I don't want to add it to this check-in.